example-config.yaml

```diff
@@ -2,10 +2,26 @@ stormpath:
   application:
     href: null
     name: null
-    
+
   web:
+
     basePath: null
```

Comment:
The `basePath` is currently used for setting the path value on cookies, and this is useful when the web application doesn’t know about the domain context, it just know’s that it’s served at `/foo` and needs to set a cookie that is local to it’s context.

The `baseDomain` is useful when, as we’ve discussed, the application needs to know how to resolve subdomains.  We want to use this for our multi-tenant features, but we also seem to think it may be useful elsewhere, and I tend to agree.

Because these two cases are pretty different, I think it’s useful to have both.